test/ctia/entity/incident_test.clj

```diff
@@ -100,13 +104,13 @@
    (fn [app]
      (helpers/set-capabilities! app "foouser" ["foogroup"] "user" all-capabilities)
      (whoami-helpers/set-whoami-response app "45c1f5e3f05d0" "foouser" "foogroup" "user")
-     (let [parameters (into sut/incident-entity
-                            {:app app
+     (let [parameters (assoc sut/incident-entity
+                             :app app
```

Comment:
`into` is actually faster than `assoc` with variadic number of arguments. Mostly because `into` is using mutable collection under the hood, and `assoc` recursively calls itself.